scripts/installer/windows/product.wxs

```diff
@@ -90,7 +90,7 @@
             ExeCommand="wscript.exe promptflow_service.vbs"
             Return="asyncNoWait" />
         <InstallExecuteSequence>
-            <Custom Action="StartPromptFlowService" Before="InstallFinalize">NOT Installed AND NOT WIX_UPGRADE_DETECTED</Custom>
+            <Custom Action="StartPromptFlowService" Before="InstallFinalize">NOT Installed OR WIX_UPGRADE_DETECTED</Custom>
```

Comment:
It may be failed to start the service because the port is used.

Reply:
I test it, it will stop pfs firstly, then uninstall old version and install new version. 
Any better ideas here to start pfs when upgrade package? Because if it remains the condition here, it seems it will stop srevice and doesn't start it when upgrade package.

Reply:
If it starts the service because the port is used, is it because the service is already started, or the service didn't stop when uninstall?

Reply:
offline sync